discord_slash/client.py

```diff
@@ -54,14 +54,31 @@ def __init__(self,
         else:
             self._discord.add_listener(self.on_socket_response)
             self.has_listener = True
+            default_add_function = self._discord.add_cog
+            def override_add_cog(cog: commands.Cog):
+                default_add_function(cog)
+                self.get_cog_commands(cog)
+            self._discord.add_cog = override_add_cog
+            default_remove_function = self._discord.remove_cog
+            def override_remove_cog(cog: commands.Cog):
```

Comment:
`remove_cog`'s parameter is `str`, not `commands.Cog`. Which means we can't remove the cog slash command with this way.